_posts/2017-10-14-eslint-v4.9.0-released.md

```diff
@@ -9,10 +9,14 @@ tags:
 
 We just pushed ESLint v4.9.0, which is a minor release upgrade of ESLint. This release adds some new features and fixes several bugs found in the previous release. 
 
+## Highlights
 
+We've added 2 new rules with this release:
 
+* [multiline-comment-style](/docs/rules/multiline-comment-style)
+* [lines-between-class-members](/docs/rules/lines-between-class-members)
 
-
+We also added new CLI option `--fix-dry-run`. This option will allow you to apply fixes without writing them to the disk.
```

Comment:
Nitpick: It would be good to link to the documentation here.

Reply:
Good point, I'll update it